install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
```

Comment:
We will need to establish the permissions of the serviceaccount

e.g.

`oadm policy add-scc-to-user privileged system:serviceaccounts:os-exploder`

or something of that variety.